### PR TITLE
NOISSUE - Add MQTT to kubernetes nginx

### PR DESCRIPTION
**Pull request by anovakovic01**
**Summary:**
- Add soft links for nginx configuration and certificates
- Add dashflux to nginx configuration for docker-compose
- Update documentation

### CODE REVIEW

File: k8s/nginx/nginx.conf [hunk 1]
```diff
@@ -0,0 +1 @@
+/Users/novke/go/src/github.com/mainflux/mainflux/docker/nginx.conf
```
Comment (drasko) [inline k8s/nginx/nginx.conf]: Hardcoded path?

File: docker/nginx.conf [hunk 2]
```diff
@@ -214,6 +214,22 @@ http {
                 return 200;
             }
         }
+
+        location / {
```
Comment (drasko) [inline docker/nginx.conf:218]: Is this thing adding MQTT or Dashflux?

File: k8s/nginx/nginx.conf (hunk 1 above)
Comment (nmarcetic) [inline k8s/nginx/nginx.conf]: ?

File: k8s/nginx/ssl [hunk 3]
```diff
@@ -0,0 +1 @@
+/Users/novke/go/src/github.com/mainflux/mainflux/docker/ssl
```
Comment (drasko) [inline k8s/nginx/ssl]: Hardcoded path?

Comment (nmarcetic) [inline k8s/nginx/ssl]: ?

**Review by drasko — CHANGES_REQUESTED** [review]

**Review by nmarcetic — CHANGES_REQUESTED** [review]
What are dose files with hard-coded local paths  ?

File: docker/nginx.conf (hunk 2 above)
Comment (anovakovic01) [reply docker/nginx.conf:218]: This is adding Dashflux, because now, compose and k8s share same configuration, and k8s already had Dashflux.

**Review by nmarcetic — APPROVED** [review]

File: docker/nginx.conf (hunk 2 above)
Comment (drasko) [reply docker/nginx.conf:218]: I see, but PR name is confusing then.

**Review by drasko — APPROVED** [review]

File: docker/nginx.conf (hunk 2 above)
Comment (anovakovic01) [reply docker/nginx.conf:218]: Point of this PR is to add MQTT to nginx for kubernetes. This change was needed as I had to share nginx configuration.

**Pull request merged**